app/controllers/crosswalk_issues_controller.rb

```diff
@@ -47,17 +47,21 @@ def find_matches
     @issue = CrosswalkIssue.find(params[:id])
     address_data_to_match = @issue.weam.address_values.join
     physical_address_data = @issue.weam.physical_address_values.join
-    query = 'similarity(institution, ?) > 0.5 OR similarity((city||state||zip||addr), ?) > 0.3' \
-            'OR similarity((city||state||zip||addr), ?) > 0.3'
     escaped_institution = ApplicationRecord.connection.quote(@issue.weam.institution)
-    sanitize_order = IpedsHd.sanitize_sql_for_order("similarity(institution, #{escaped_institution}) DESC,
-                                         similarity((city||state||zip||addr), '#{address_data_to_match}') DESC,
-                                         similarity((city||state||zip||addr), '#{physical_address_data}') DESC")
 
-    @ipeds_hd_arr = IpedsHd.where(query, "%#{@issue.weam.institution}%",
-                                  "%#{address_data_to_match}%",
-                                  "%#{physical_address_data}%")
-                           .order(sanitize_order)
+    str = <<-SQL
+        SELECT *, GREATEST(similarity(institution, #{escaped_institution}),
```

Comment:
I should have mentioned this on my last review. Please explicitly list the fields for the select instead of using the asterisk. Then it will be good to go.

Reply:
Got it. The changes are in along with removing the pointless mapping method.